app/src/test/java/nerd/tuxmobil/fahrplan/congress/utils/ConferenceTimeFrameTest.kt

```diff
@@ -48,59 +48,59 @@ class ConferenceTimeFrameTest {
     }
 
     @Test
-    fun firstDayStart() {
+    fun `firstDayStart returns the the first day`() {
         assertThat(conference.firstDayStartTime).isEqualTo(FIRST_DAY_START_TIME)
     }
 
     @Test
-    fun containsWithTimeWithFirstDaySession() {
+    fun `contains returns true if time marks a session at the first day`() {
         // 2015-12-27T11:30:00+0100, in seconds: 1451212200000
         assertThat(conference.contains(1451212200000L)).isTrue()
     }
 
     @Test
-    fun containsWithOneSecondBeforeFirstDay() {
+    fun `contains returns false if time marks a session starting one second before the first day`() {
         // 2015-12-26T23:59:59+0100, in seconds: 1451170799000
         assertThat(conference.contains(1451170799000L)).isFalse()
     }
 
     @Test
-    fun containsWithTimeOfLastDay() {
+    fun `contains returns false if time marks a session at the last day`() {
```

Comment:
at the last day end time
(also for all the others below the same... )

Reply:
:heavy_check_mark: Thank you. Fixed in d13848f51f4f4d92e8fd463fbccee6ced1808fb3.